lists/BF-MATD

```diff
@@ -0,0 +1,55 @@
+{
+  "name": {
+    "en": "Ministry of Territorial Administration and Decentralisation",
+    "local": "Ministère de l'Administration Territoriale et de la Décentralisation"
+  },
+  "url": "https://www.matd.gov.bf/accueil",
+  "description": {
+    "en": "The Ministry of Territorial Administration and Decentralisation is in charge of the organisation and administration of the national territory and the management of decentralisation. Civil society organizations (CSOs) are able to form freely and can seek and secure registration by submitting declarations of their existence along with draft bylaws, internal regulations, and other information to the Ministry of Territorial Administration and Decentralization (MATD). \n"
+  },
+  "coverage": [
+    "BF"
+  ],
+  "structure": [
+    "government_agency"
```

Comment:
This list seems to cover CSOs, not government departments and ministries. So I think this value should be 'charity'